freshbooks/api/business.py

```diff
@@ -0,0 +1,181 @@
+from decimal import Decimal
```

Comment:
@amcintosh would you want to rename this to accounting_business.py as well?

Reply:
Yes please.

Reply:
Made this change